Package.swift

```diff
@@ -14,9 +14,11 @@ let package = Package(
     ],
     dependencies: [
         .package(
-          url: "https://github.com/apple/swift-collections",
-          revision: "2ca40e2a653e5e04a1c5468a35fc7494ef6db1d3"), // on release/1.1
-        .package(url: "https://github.com/apple/swift-foundation-icu", exact: "0.0.2")
+            url: "https://github.com/apple/swift-collections",
+            revision: "2ca40e2a653e5e04a1c5468a35fc7494ef6db1d3"), // on release/1.1
+        .package(
+            url: "https://github.com/apple/swift-foundation-icu",
+            revision: "0c1de7149a39a9ff82d4db66234dec587b30a3ad")
```

Comment:
Can you add a comment about the significance of this hash (similar to the one you added above for swift-collection)?